samples/dashboards/demo/live-weather/demo.js

```diff
@@ -0,0 +1,969 @@
+/* eslint-disable jsdoc/require-description */
+
+// Index of the data to be displayed in map, KPI and spline chart.
+// The number is an offset from the current hour.
+
+const rangeConfig = {
+    first: 0, // 0: current observation
+    hours: 24 // max: 19 days
+};
+
+// Parameter configuration
+const paramConfig = {
+    temperature: {
+        name: 'temperature',
+        unit: '˚C',
+        min: -20,
+        max: 50,
+        colorStops: [
+            [0.0, '#4CAFFE'],
+            [0.3, '#53BB6C'],
+            [0.5, '#DDCE16'],
+            [0.6, '#DF7642'],
+            [0.7, '#DD2323']
+        ]
+    },
+    pressure: {
+        name: 'pressure',
+        unit: 'hPa',
+        min: 800,
+        max: 1100,
+        colorStops: [
+            [0.0, '#000000'],
+            [0.5, '#CCCCCC'], // 950 hPa
+            [0.7, '#6666CC'], // 1013 hPa
+            [0.8, '#3333CC']  // 1030 hPa
+        ]
+    },
+    humidity: {
+        name: 'humidity',
+        unit: '%',
+        min: 0,
+        max: 100,
+        colorStops: [
+            [0.4, '#dceff5'],
+            [0.5, '#87abd6'],
+            [0.6, '#dceff5'],
+            [0.7, '#5470b3'],
+            [0.8, '#2843b8'],
+            [0.9, '#0f11a6']
+        ]
+    },
+    buildDescription: function (param) {
+        const info = this[param];
+        // First letter uppercase
+        const name = info.name[0].toUpperCase() + info.name.slice(1);
+
+        return `${name} (${info.unit})`;
+    }
+};
+
+// Geolocation info: https://www.maps.ie/coordinates.html
+
+// Three sets of locations are available:
+//  - worldLocations, testLocations, euroLocations
+//  - selected in "weatherStations.location"
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
We are using the `worldLocations` so there is no need to disable this. 
But if there is no dynamic toggle between the sets I would remove the rest.

Reply:
I was in two minds here; remove the euro and test data sets, or add a dropdown to select one of them. "Cheapest" option is to leave only the world set, perhaps with a few extra cities.

Reply:
> We are using the `worldLocations` so there is no need to disable this. But if there is no dynamic toggle between the sets I would remove the rest.

Removed.